readme.md

```diff
@@ -103,26 +103,29 @@ npx truffle migrate
 yarn test
 ```
 
+### Lint
 
-### Break down into end to end tests
+ESLint will be run automatically before you commit. To run it manually:
 
-Explain what these tests test and why
 
 ```
-Give an example
+yarn lint:fix
 ```
 
-### And coding style tests
+## Deployment
 
-Explain what these tests test and why
+The code is deployed automatically on each push to GitHub. A GitHub action will create a test website and post the link as a comment in the PR.
 
-```
-Give an example
-```
+When pushing to the `master` branch, the code will be automatically deployed to production.
 
-## Deployment
+## Releasing to production
+
+We prepare a new release every sprint.
```

Comment:
Probably worth adding how long the spring is, otherwise it doesn't give much value IMO